kubernetes/cmsweb/scripts/deploy-cm.sh

```diff
@@ -21,19 +39,33 @@ conf=$3
     fi
 
 
+    sopskey=$SOPS_AGE_KEY_FILE
+    kubectl get secrets $ns-keys-secrets -n $ns --template="{{index .data \"$ns-keys.txt\" | base64decode}}" > "$ns-keys.txt"
```

Comment:
Originally, I think you forgot to put it into `$tmpDir/$ns-keys.txt` but then I look-up the code and saw that you cd into tmp area. Still I would prefer to have `$tmpDir/$ns-keys.txt` explicitly specified here.